### ISSUE TITLE
Integration of plugin with both ios and android

### ISSUE THREAD

**Issue by sangariv** [m1is/AviaryCordovaPlugin#10]
HI,
DO i need to create separate api-key for android an ios to develop?
can u pls reply me?

**Comment by m1is** [comment]
No, the same API key will work with all their versions, Android, iOS and Web.

**Comment by sangariv** [comment]
HI,

Thanks for ur reply.
when i use your code,watermark of aviary is shown.is there anyway to ged rid of it

**Comment by m1is** [comment]
There is a way to remove it but you would need to get a paid version to do that.
